modules/mco-update-boot-images-disable.adoc

```diff
@@ -9,6 +9,21 @@
 
 To disable the updated boot image feature, edit the `MachineConfiguration` object so that the `machineManagers` field is an empty array.
 
+[NOTE]
+====
+If you are upgrading a Google Cloud Platform (GCP) or Amazon Web Services (AWS) cluster from {product-title} 4.18 to 4.19, and you have not configured the `managedBootImages` parameter, the upgrade is blocked with the _This cluster is GCP or AWS but lacks a boot image configuration._ message. The update is blocked intentionally in order to alert you that the default updated boot image behavior is changing between version 4.18 and 4.19 to enable updated boot images by default.
+
+To restart the update, perform one of the following tasks:
+
+* If you do not want the updated boot image feature enabled, explicitly disable the feature as described in this section.
+* If you want to allow the feature to be enabled, acknowledge that you are aware of the change in default behavior by patching the `admin-acks` config map by using the following command:
```

Comment:
```suggestion
* If you want to enable the feature, patch the `admin-acks` config map by using the following command:
```

I think "acknowledge that you are aware of the change in default behavior" is already mentioned earlier in the bigger paragraph.

Reply:
1. The user isn't enabling the feature, it will be enabled upon upgrade if you don't explicitly disable it.  
"If you want the feature enabled"??

2. The developers and I just wanted to be explicit about what the admin-ack command is for; it is an acknowledgement that you  are aware that this change is taking place.